apps/vaporgui/MainForm.cpp

```diff
@@ -879,7 +853,7 @@ void MainForm::sessionNew()
     _paramsMgr->UndoRedoClear();
 
     _stateChangeFlag = false;
-    _sessionNewFlag = true;
+    GetStateParams()->SetValueLong(GUIStateParams::SessionNewTag, "Toggle SessionNewTag to true for a new session", true);
```

Comment:
Please double check that extraneous renders are not triggered by the occurrences of replacing _sessionNewFlag by writing to the params database

Reply:
No problem.  The following sequence of events results in 3 calls to MainForm::Render on our main branch as well as this one, issue3778.

1) Launch VAPOR
2) Import any WRF file
3) Create new session

So, no extraneous renders found.